api/apps/api/src/modules/users/users.controller.ts

```diff
@@ -49,6 +52,9 @@ import { UpdateUserPasswordDTO } from './dto/update.user-password';
 import { IsMissingAclImplementation } from '@marxan-api/decorators/acl.decorator';
 import { PlatformAdminEntity } from './platform-admin/admin.api.entity';
 import { isLeft } from 'fp-ts/lib/Either';
+import { Writable } from 'utility-types';
```

Comment:
unused import, please remove (also: it would be `import { Writable } from 'stream';` for this use case)